.storybook/addons.js

```diff
@@ -5,12 +5,9 @@ import '@storybook/addon-backgrounds/register';
 import '@storybook/addon-viewport/register';
 import '@storybook/addon-a11y/register';
 import '@storybook/addon-storysource/register';
-import { registerDsm } from '@invisionapp/dsm-storybook/register';
 import { addons } from '@storybook/addons';
 import canopyTheme from './canopy-theme';
 
 addons.setConfig({
   theme: canopyTheme,
 });
-
-registerDsm(process.env.STORYBOOK_DSM);
```

Comment:
cool i guess this gets cleaned up manually after this branch merges :-)

Reply:
as in removing the secret

Reply:
we never put the secret in, it was only there for if you ran it locally.